### DISCUSSION TITLE
How can i always show a marker (dot)  for every value in a line chart?

### DISCUSSION THREAD

**Discussion by Sok4h** [patrykandpatrick/vico#863] [Questions]
### Question

Hi, first of all, thanks for this library.  I'm stuck figuring out how to always show a dot in each data point and show the label only when a value is pressed. I would really appreciate some guidance on how to achieve this

![image](https://github.com/user-attachments/assets/5179c3ac-12b1-45f9-b04d-8841970eaded)

I'm sorry if the question is too simple. thanks

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Sok4h** [comment]
nvm i figure it out. really cool library thanks

**Reply by fangedrabbit** [reply]
Can you post your solution (explanation, code, and/or screenshot) if possible?

**Reply by Praveen-Android** [reply]
I answered it here: https://github.com/patrykandpatrick/vico/discussions/874#discussioncomment-10634192. Something like

```
rememberLine(
            fill = remember { LineCartesianLayer.LineFill.single(fill(Color.Blue)) },
            pointProvider = LineCartesianLayer.PointProvider.single(
              LineCartesianLayer.Point(
                component = rememberShapeComponent(
                  component = rememberShapeComponent(Color.Blue, Shape.Pill),
                  ...
                )
              )
            )
         )
```

**Reply by patrickmichalik** [reply]
I’m glad to hear that you’ve found the solution, @Sok4h. Thanks for the kind words. @Praveen-Android, thanks for the help. @fangedrabbit, if you have any further questions, feel free to ask.
